modules/references/client/components/Interaction.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Interaction(props) {
+
+  const { reference: { interactions } } = props;
+  const isInteraction = [...Object.keys(interactions)].reduce((accumulator, current) => accumulator || interactions[current], false);
+  return (
+    <div className="panel panel-default">
+      <div className="panel-heading">
+        <h4 id="howDoYouKnowThemQuestion">How do you know them?</h4>
+      </div>
+      <div className="panel-body">
+        <div role="group" aria-labelledby="howDoYouKnowThemQuestion">
+          <div className="checkbox">
+            <label>
+              <input
+                type="checkbox"
+                checked={props.reference.interactions.met}
+                onChange={() => props.onChange('met')}
+              />
+              Met in person
+            </label>
+          </div>
+          <div className="checkbox">
+            <label>
+              <input
+                type="checkbox"
+                checked={props.reference.interactions.hostedThem}
+                onChange={() => props.onChange('hostedThem')}
+              />
+              I hosted them
+            </label>
+          </div>
+          <div className="checkbox">
+            <label>
+              <input
+                type="checkbox"
+                checked={props.reference.interactions.hostedMe}
+                onChange={() => props.onChange('hostedMe')}
+              />
+              They hosted me
+            </label>
+          </div>
+        </div>
+        {(!isInteraction) ? (
+          <div className="alert alert-warning reference-new-tabs-alert" role="alert">
```

Comment:
Let's remove this badge, it feels like something was broken and it's already obvious from the page without it that user should make a choice at this page:

<img width="431" alt="image" src="https://user-images.githubusercontent.com/87168/50034112-01b4b500-0004-11e9-8acd-087a417648b5.png">

Reply:
I guess it was originally meant to be there if user comes back to this view using "back" button and then tries to press next without choosing anything from here?

Reply:
Removed badge from Interaction and Recommendation components in 85bb82bce1e976251efe4811c6fdc97ee9a096e3.